src/home_robot/home_robot/agent/ovmm_agent/ovmm_agent.py

```diff
@@ -111,6 +239,129 @@ def _modular_nav(self, obs: Observations) -> Tuple[DiscreteNavigationAction, Any
             self._switch_to_next_skill(e=0)
         return action, info
 
+    def _modular_place(self, obs: Observations):
+        """
+        1. Get estimate of point on receptacle to place object on.
+        2. Orient towards it.
+        3. Move forward to get close to it.
+        4. Rotate 90º to have arm face the object. Then rotate camera to face arm.
+        5. (again) Get estimate of point on receptacle to place object on.
+        6. With camera, arm, and object (hopefully) aligned, set arm lift and
+        extension based on point estimate from 4.
+        """
+        RETRACTED_ARM_LENGTH = 0.2
+        GRIPPER_LENGTH = 0.12
```

Comment:
i believe this value is already in here: https://github.com/facebookresearch/home-robot/blob/main/src/home_robot/home_robot/motion/stretch.py#L87

but this looks different

Reply:
Updated. Pushing soon.